### DISCUSSION TITLE
True Wireless Stereo

### DISCUSSION THREAD

**Discussion by tv4you2016** [schreibfaul1/ESP32-audioI2S#689] [Ideas]
hello
It would be funny if we could use the lib with the possibility of True Wireless Stereo
I found this project but in BT can you adapt it to use your lib?

https://github.com/RASPIAUDIO/LRAudioNow

**Comment by schreibfaul1** [comment]
The problem is that WiFi and BT do not work simultaneously on an ESP32. What you can do is to stream from an audio source like SD card via BT. [ESPuino](https://github.com/biologist79/ESPuino) does this. Another option is to use an external BT device, such as [KCX BT Emitter](https://github.com/schreibfaul1/ESP32-KCX-BT-EMITTER), where sending and receiving is possible in good quality, but without metadata
